packages/core/src/common/command.ts

```diff
@@ -348,6 +348,15 @@ export class CommandRegistry implements CommandService {
         return undefined;
     }
 
+    /**
+     * Returns with all handlers for the given command. If the command does have any handlers,
```

Comment:
I believe there is a typo? 
"If the command **does not** have any handlers..."

Reply:
Eagle eye ;) Fixed it, thanks!